models/migrations/v24.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 Gitea. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"github.com/go-xorm/xorm"
+)
+
+type StatusTable struct {
+	ID          int64  `xorm:"pk autoincr"`
+	Index       int64  `xorm:"INDEX UNIQUE(repo_sha_index)"`
```

Comment:
It is the Counter across Repo and SHA. Each SHA has up to 1000 statuses (I don't actually limit, github api docs specifies it though).